projects/core/components/textfield/textfield.component.ts

```diff
@@ -171,7 +171,7 @@ export class TuiTextfieldBaseComponent<T>
 
         if (
             !this.dropdownOpen.tuiDropdownEnabled ||
-            !this.cva?.interactive() ||
+            !(this.cva?.interactive() ?? this.control?.enabled) ||
```

Comment:
Do we even need this line? Wouldn't the next line cover this case? 🤔

Reply:
Before:

`this.cva` is undefined -> `this.cva?.interactive()` ->  `!(undefined)` -> if (true)


After:

`this.cva ?? this.control` -> is (undefined ?? {enabled: true}) ->  `!(undefined ?? true)` -> if (false)

Reply:
I mean if it's not interactive then next line would match anyway, I think we can just remove this line altogether.

Reply:
However it's interactive 🤔 or I don't understand what is mean interactive mode

<img width="502" height="455" alt="image" src="https://github.com/user-attachments/assets/4ab1e2a8-443f-4e32-b047-d31477d2face" />